core/src/main/java/com/radixdlt/p2p/addressbook/BerkeleyAddressBookStore.java

```diff
@@ -101,7 +101,7 @@ public final class BerkeleyAddressBookStore implements AddressBookPersistence {
   public BerkeleyAddressBookStore(
       Serialization serialization,
       Metrics metrics,
-      @NodeStorageLocation String nodeStorageLocation,
+      @StateManagerStorageLocation String nodeStorageLocation,
```

Comment:
I don't understand this - We're effectively moving the Berkeley address book here? 

I'd assumed we only want this for the purposes of migrating to the new address book store? So it should use the old location?

Reply:
This is to preserve the Berkeley address book location where it was before. It might be a bit misleading as the old `NodeStorageLocation` annotation is renamed to `StateManagerStorageLocation` and the new `NodeStorageLocation` annotation is now responsible for node-specific data - address book and safety state.

Reply:
Restored old naming